common/browser_context_test.go

```diff
@@ -134,13 +136,16 @@ func TestFilterCookies(t *testing.T) {
 				"https://sub.foo.com",
 			},
 			cookies: []*api.Cookie{
+				{
+					Domain: "sub.foo.com",
+				},
 				{
 					Domain: ".foo.com",
```

Comment:
Oh, now `.foo.com` is being filtered out, is that correct? It wasn't being filtered out in the previous iteration. I'm not 100% clued up on this so i'm mainly asking asking what's the expected result 🙂

Reply:
I've noticed when the subdomain is explicitly given to filter cookies with subdomains, then it should do so :)